course_discovery/apps/api/tests/test_serializers.py

```diff
@@ -1059,6 +1083,22 @@ def test_data(self):
         serializer = ContainedCourseRunsSerializer(instance)
         self.assertDictEqual(serializer.data, instance)
 
+    def test_contained_course_runs_filtered_on_edx_org_short_name(self):
+        """
+        Verify course runs keys filtered on elastic search query string & edX organization.
+        """
+        edx_org_short_name = 'edX'
+        serializer = self.serializer_class.prefetch_queryset(
```

Comment:
Please try to avoid any generic variable/method names like `serializer`, `expected` here. Update these variables.

Reply:
Done